homeassistant/components/command_line/__init__.py

```diff
@@ -2,49 +2,75 @@
 from __future__ import annotations
 
 import logging
-import subprocess
+from typing import Any
+
+from homeassistant.config_entries import SOURCE_IMPORT, ConfigEntry
+from homeassistant.const import CONF_PLATFORM, Platform
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers import discovery
+from homeassistant.helpers.typing import ConfigType
+
+from .const import DOMAIN
 
 _LOGGER = logging.getLogger(__name__)
 
 
-def call_shell_with_timeout(
-    command: str, timeout: int, *, log_return_code: bool = True
-) -> int:
-    """Run a shell command with a timeout.
+async def async_setup(hass: HomeAssistant, config: ConfigType) -> bool:
+    """Set up the Command Line component to import config."""
+    # Iterate all entries for notify to only get Discord
+    if Platform.NOTIFY in config:
```

Comment:
You could save some indent if you make this a guard and return True when the conditional is reversed